resources/views/includes/footer.blade.php

```diff
@@ -1,15 +1,15 @@
-<footer class="p-0 fixed-bottom">
+<footer- class="p-0 fixed-bottom">
     <ul class="nav nav-pills navbar-dark d-flex flex-row text-center justify-content-around">
         <li class="nav-item">
             <a class="nav-link" data-toggle="tab" href="{{route('home')}}">Home</a> </li>
         <li class="nav-item">
             <a class="nav-link selected" data-toggle="tab" href="{{route('lists.index')}}"> Lists</a>
         </li>
         <li class="nav-item">
-            <a class="nav-link" data-toggle="tab" href="#"> Songs</a>
+            <a class="nav-link" data-toggle="tab" href="{{route('songs.index')}}"> Songs</a>
         </li>
         <li class="nav-item">
             <a class="nav-link" data-toggle="tab" href="#"> Tools</a>
```

Comment:
Is that space in front of "Tools" or "Songs" there on purpose?

Reply:
You found the whitespaces that were missing in other places! Thanks!